WebAPI/Repository/BetRepository.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using WebAPI.Model;
+using WebAPI.Model.Entities;
+
+namespace WebAPI.Repository
+{
+    public class BetRepository : IBetRepository
+    {
+        private readonly EzBetDbContext ezBetDbContext;
```

Comment:
I suggest to name private readonly variables with starting _ sign. I mean renaming `ezBetDbContext` to `_ezBetDbContext`.